src/node_url.cc

```diff
@@ -360,13 +360,11 @@ void BindingData::CreatePerIsolateProperties(IsolateData* isolate_data,
   SetMethodNoSideEffect(isolate, target, "getOrigin", GetOrigin);
   SetMethod(isolate, target, "parse", Parse);
   SetMethod(isolate, target, "update", Update);
-  SetFastMethodNoSideEffect(
-      isolate, target, "canParse", CanParse, &fast_can_parse_);
-  SetFastMethodNoSideEffect(isolate,
-                            target,
-                            "canParseWithBase",
-                            CanParse,
-                            &fast_can_parse_with_base_);
+
+  const v8::CFunction fast_can_parse_methods[] = {fast_can_parse_,
```

Comment:
I am not sure if it's guaranteed to be safe to copy `v8::CFunction`s, but we might as well just move this to the outer scope and initialize the methods in the array's initializer list.

Reply:
Might be off-topic: if it's not safe to be copied, should its implicit copy constructor be deleted in v8?

Reply:
I've updated the pr. Appreciate reviewing again